### PR TITLE
Removed repeated words and added more nouns

### PR DESCRIPTION
**Pull request by pankaj892**

### CODE REVIEW

File: words_dir/tweet_nouns.py [hunk 1]
```diff
@@ -201,16 +201,22 @@
...
     'tea',
-    'dahi',
+    'yoghurt',
+    'coffee',
+    'gulab jamun',
+    'Lassi',
+    'Khichdi',
+    'Lemonade',
+    'Pulao',
+    'Kofta'
```
Comment (bobsany16) [inline words_dir/tweet_nouns.py]: missing comma

File: words_dir/tweet_nouns.py [hunk 2]
```diff
@@ -342,7 +353,12 @@
     'calculator',
     'valley',
     'lake',
-    'moonlit sky',
+    'Chalk',
+    'Duster',
+    'Bench',
+    'Stencil',
+    'Protractor',
+    'Compass'
```
Comment (bobsany16) [inline words_dir/tweet_nouns.py]: comma

File: words_dir/tweet_nouns.py [hunk 3]
```diff
@@ -443,6 +459,8 @@
     'tampon',
     'bodywash',
     'foot massager',
+    'mirror',
+    '
```
Comment (bobsany16) [inline words_dir/tweet_nouns.py]: the quotation mark should be fixed

File: words_dir/tweet_nouns.py (hunk 1 above)
Comment (pankaj892) [reply words_dir/tweet_nouns.py]: I have fixed them check

**Review by VickiLanger — APPROVED** [review]
Thanks for your contribution.

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Pull request merged**